topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: tutorial_hands_on
+
+title: "NCBI BLAST+ against the MAdLand"
+zenodo_link: 'https://zenodo.org/record/4710649'
+questions:
+- "How to access Galaxy server?"
```

Comment:
In theory users should know how to access the server, so perhaps this question is not required.

Reply:
noted. I will delete it